src/dom/attributes/toggles/hide-if-attr.js

```diff
@@ -1,3 +1,22 @@
+/**
+ * ## Display Elements Conditionally (hideif)
+ *
+ * The `data-f-hideif` attribute allows you to hide DOM elements based on either the value of the model variable (true or false), or the value of a [comparison (Number Comparison Converter)](../../../../converters/number-compare-converter/) using a model variable.
+ *
+ * **Examples:**
+ *
+ *      <!-- model variable already has a boolean value -->
+ *      <div data-f-hideif="sampleBooleanModelVariable">Remains hidden if the model variable is true</div>
+ *
+ *      <!-- chain with the greaterThan converter to produce a boolean value, 
```

Comment:
The equivalent of the above would be piping to `toBool`; but this example is useful as-is so we should keep all 3